test/performance/shmem_perf_suite/common.h

```diff
@@ -111,23 +111,30 @@ typedef enum {
 } bw_units;
 
 typedef enum {
-    OP_FETCH,
     OP_SET,
+    OP_INC,
+    OP_ADD,
+    OP_AND,
+    OP_OR,
+    OP_XOR,
+    OP_FETCH,
     OP_CSWAP,
     OP_SWAP,
     OP_FINC,
-    OP_INC,
     OP_FADD,
-    OP_ADD,
     OP_FAND,
-    OP_AND,
     OP_FOR,
-    OP_OR,
     OP_FXOR,
-    OP_XOR,
-    SIZE_OF_OP
+    LAST_OP
 } atomic_op_type;
 
+#define FIRST_OP OP_SET
```

Comment:
This shouldn't be necessary, C guarantees that the first entry in the enumeration is `0` unless you say otherwise.

Reply:
Yes. I was just keeping an alias.. I will drop it then.